src/Transfer.php

```diff
@@ -2302,7 +2302,7 @@ public function transferContracts($itemtype, $ID, $newID)
 
                         if (count($contract_iterator)) {
                             $result = $iterator->current();
-                            $newcontractID = $result['id'];
+                            $newcontractID = $result['contracts_id'];
```

Comment:
How can this work? Because L2295, SELECT does not have `contracts_id`.

Reply:
Yes indeed, I saw that but forgot to mention. This PR seems incorrect.